README.md

```diff
@@ -89,40 +95,40 @@ The supported hashing algorithms are `spooky` and `sha256`.
 
 ### Create a new Database Entry and upload
 
-Create an new EDM DB entry and upload the associated file.
+Create an new EDM DB entry and upload the associated file. The file has to be hashed prior to creating the database entry.
 
 ```bash
-edmtool create_and_upload --name "Your DB Name" --description "Your Description" --metadata_file_path /path/to/your/file_encoded_metadata.json --base_url http://example.cyberhaven.com --token YOUR_AUTH_TOKEN
+edmtool create_and_upload --name "Your DB Name" --description "Your Description" --metadata_file_path /path/to/your/file_encoded_metadata.json --base_url https://api.cyberhaven.com --token YOUR_AUTH_TOKEN
 ```
 
 ### Update an existing Database Entry and upload
 
-Update an existing EDM DB entry and upload the associated file.
+Update an existing EDM DB entry and upload updated associated file. The file has to be hashed prior to updating the database entry.
 
 ```bash
-edmtool update_and_upload --id DATABASE_ID --description "Your Description" --metadata_file_path /path/to/your/updated_file_encoded_metadata.json --base_url http://example.cyberhaven.com --token YOUR_AUTH_TOKEN
+edmtool update_and_upload --id DATABASE_ID --description "Your Description" --metadata_file_path /path/to/your/updated_file_encoded_metadata.json --base_url https://api.cyberhaven.com --token YOUR_AUTH_TOKEN
```

Comment:
we use cyberhaven.io instead of cyberhaven.com for customer environments. Perhaps replace api.customer.com with your-tenant.cyberhaven.io
The rest looks good to me.